MoltenVK/MoltenVK/API/mvk_private_api.h

```diff
@@ -241,7 +241,8 @@ typedef struct {
 	VkBool32 shouldMaximizeConcurrentCompilation;                              /**< MVK_CONFIG_SHOULD_MAXIMIZE_CONCURRENT_COMPILATION */
 	float timestampPeriodLowPassAlpha;                                         /**< MVK_CONFIG_TIMESTAMP_PERIOD_LOWPASS_ALPHA */
 	VkBool32 useMetalPrivateAPI;                                               /**< MVK_CONFIG_USE_METAL_PRIVATE_API */
-	uint32_t _unused_struct_padding;
+	VkBool32 useMetalNativeTextureAtomics;                                     /**< MVK_CONFIG_USE_METAL_NATIVE_TEXTURE_ATOMICS */
```

Comment:
What is the need for a config parameter? Are there use cases where the app would want to override `MVKPhysicalDeviceMetalFeatures::nativeTextureAtomics` for some reason?

Reply:
I guess I was being overly tentative about possible cases which would be supported by the emulation, but not natively. I mostly had cubes in mind, but they're not supported either way.. I got rid of the parameter.